pinery-client/src/main/java/ca/on/oicr/pinery/client/ProjectClient.java

```diff
@@ -17,5 +17,6 @@ public ProjectClient(PineryClient mainClient) {
 	public List<SampleProjectDto> all() throws HttpResponseException {
 		return getResourceList("sample/projects");
 	}
-	
+    //TODO
+    //FIXME	
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Take the required action to fix the issue indicated by this comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1134)